notebooks/13-Case-Study-2-Solutions.ipynb

```diff
@@ -471,18 +551,41 @@
    "cell_type": "code",
    "execution_count": 55,
    "id": "eed9bd14",
-   "metadata": {},
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
    "outputs": [],
    "source": [
     "heart.fillna({\"ca\": 0.0, \"thal\": \"normal\"})\n",
```

Comment:
Is there a reason to do `fillna` twice? Don't we just want the second line?

This whole solution does seem bit overwhelming though, but I'm not sure how easy it would be to break it into more understandable chunks.

Reply:
We used it mostly as a more manual alternative, but we can clean it up and just mention it while running through it.